### DISCUSSION TITLE
Request for Source Code of SNSched Scheduler in Your Paper

### DISCUSSION THREAD

**Discussion by Gxz0128** [anujpathania/HotSniper#53] [General]
Dear Anuj,

I hope this email finds you well. My name is Jia Xi, and I am a graduate student currently pursuing research about S-NUCA Many-Cores.

I am reaching out to you regarding your paper titled [**_Task Scheduling for Many-Cores with S-NUCA Caches_**], published in Date on 2018. Your research on S-NUCA Many-Cores has been instrumental in shaping my understanding of the field.

I am particularly interested in the _SNSched_ scheduler discussed in your paper. As a graduate student delving deeper into S-NUCA Many-Cores, I believe that exploring the implementation details of SNSched would greatly contribute to my current research endeavors.

Hence, I would like to kindly request your assistance in obtaining the source code of the _SNSched_ scheduler. Access to this code would not only deepen my comprehension of the concepts elucidated in your paper but also facilitate further experimentation and analysis, crucial for advancing my research.

If feasible, could you please share the source code with me or guide me to the appropriate resources where I can access it?

Your support in this matter would be immensely valuable to me, and I truly appreciate your consideration of my request. Please let me know if there are any formalities or additional steps required from my end to obtain the source code.

Thank you for your time and attention. I eagerly await your response.

Warm regards,
Jia Xi

**Comment by anujpathania** [comment]
Hi Jia Xi,

I am afraid, I didn't keep the code for SNSched. Also, SNSched is quite outdated now.

I recommend looking at our latest paper on SNUCA called "Thermal Management for S-NUCA Many-Cores via Synchronous Thread Rotation". The code for it is available here - https://github.com/yixianuva/hotpotato

Regards,
Anuj Pathania
